Cannot find module './lodash-id.js

```
Error: Cannot find module './lodash-id.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at LocalSession._initDB (/home/amao/Git/R6statsTGbot/node_modules/telegraf-session-local/lib/session.js:274:19)
    at new LocalSession (/home/amao/Git/R6statsTGbot/node_modules/telegraf-session-local/lib/session.js:84:15)
    at Object.<anonymous> (/home/amao/Git/R6statsTGbot/index.js:19:10)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
```
The break line in my file is
```nodejs
bot.use((new LocalSession({ database: 'example_db.json' })).middleware())
```

I installed the lodash-id by npm but not solve the problem
 
Arch Linux + zsh
Webstorm

--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "node": ">=8"
   },
   "files": [
-    "lib/session.js"
+    "lib"
   ],
   "scripts": {
     "lint": "eslint . --fix",

